packages/btcindexer/src/btcindexer.test.ts

```diff
@@ -73,6 +73,12 @@ const mkMockEnv = () =>
 		DB: mkMockD1(),
 		btc_blocks: {},
 		nbtc_txs: {},
+		SUI_NETWORK: "testnet",
+		SUI_PACKAGE_ID: "0xPACKAGE",
+		NBTC_OBJECT_ID: "0xNBTC",
+		LIGHT_CLIENT_OBJECT_ID: "0xLIGHTCLIENT",
+		SUI_SIGNER_MNEMONIC:
```

Comment:
Should this be lowercase? If this is a shell env, then probably this should be in a sub object?